src/rl_framework/util/saving_and_loading/connector/clearml_connector.py

```diff
@@ -21,11 +21,15 @@ class ClearMLUploadConfig(UploadConfig):
     n_eval_episodes (int): Number of episodes for agent evaluation to compute evaluation metrics
     video_length (int): Length of video in frames (which should be generated and uploaded to the connector).
         No video is uploaded if length is 0 or negative.
+    project_name (str): Project path in which the experiment run(s) will be created
+    task_name (Optional str): Name of experiment, None to use the script name
     """
 
     file_name: str
     n_eval_episodes: int
     video_length: int
+    project_name: str
```

Comment:
Should we make this optional as well to not break the existing interface?

Reply:
I actually removed the default again, but a generic one like "default" probably would not hurt. On the other side, ClearML itself does not define a default. So every implementation HAS to define their own default or fallback.